ts/screens/modal/RegionServiceModal.tsx

```diff
@@ -0,0 +1,191 @@
+     * this value and domain value on cookie MUST match
+     * this parameter is considered only by Android.
+     */
+    CookieManager.set("http://192.168.1.20", cookie).catch(_ =>
+      showToast(
+        `Unable to execute cookie authentication for ${cookie.name}`,
+        "danger"
+      )
+    );
+
+    if (navigationURI === "") {
+      setNavigationUri("http://192.168.1.20:3000/test-html");
+    }
+  });
+
+  const showSuccessContent = () => (
+    <Content>
+      <IconFont
+        name={"io-close"}
+        style={{
+          alignSelf: "flex-end"
+        }}
+        onPress={() => setSuccess(false)}
+      />
+      <IconFont
+        name={"io-complete"}
+        size={120}
+        color={customVariables.brandHighlight}
+        style={{
+          alignSelf: "center"
+        }}
+      />
+      <View spacer={true} />
+
+      <Text alignCenter={true}>{`${I18n.t("global.genericThanks")},`}</Text>
+      <Text alignCenter={true} bold={true}>
+        {text}
+      </Text>
+    </Content>
+  );
+
+  const showErrorContent = () => (
+    <Content>
+      <IconFont
+        name={"io-close"}
+        style={{
+          alignSelf: "flex-end"
+        }}
+        onPress={() => setError(false)}
+      />
+      <IconFont
+        name={"io-error"}
+        size={120}
+        color={customVariables.brandDanger}
+        style={{
+          alignSelf: "center"
+        }}
+      />
+      <View spacer={true} />
+
+      <Text alignCenter={true} bold={true}>
+        {text}
+      </Text>
+    </Content>
+  );
+
+  const handleWebviewMessage = (event: WebViewMessageEvent) => {
+    setWebMessage(event.nativeEvent.data);
+
+    const data = JSON.parse(event.nativeEvent.data);
+    const locale = getCurrentLocale();
+
+    switch (data.type) {
+      case "CLOSE_MODAL":
+        props.onModalClose();
+        return;
+      case "START_LOAD":
+        setLoading(true);
+        return;
+      case "END_LOAD":
+        setLoading(false);
+        return;
+      case "SHOW_SUCCESS":
+        setSuccess(true);
+        setText(data.payload[locale]);
+        return;
+      case "SHOW_ERROR":
+        setError(true);
+        setText(data.payload[locale]);
+        return;
+      case "SHOW_ALERT":
+        Alert.alert(
+          data.payload[locale].title,
+          data.payload[locale].description
+        );
+        return;
+      default:
+        return;
+    }
+  };
+
+  return (
+    <Container>
+      <AppHeader>
+        <Left>
+          <ButtonDefaultOpacity
+            onPress={() => props.onModalClose()}
+            transparent={true}
+          >
+            <IconFont name="io-back" />
+          </ButtonDefaultOpacity>
+        </Left>
+        <Body />
+      </AppHeader>
+      <View style={{ paddingHorizontal: customVariables.contentPadding }}>
+        <TextInput
+          style={{ padding: 1, borderWidth: 1, height: 30 }}
```

Comment:
```suggestion
          style={{ padding: 1, borderWidth: 1, height: 30 }}
```
styles should be declared on top and referred from components (few other cases in this file)